ui/browser/store/store.js

```diff
@@ -13,6 +13,14 @@ specific language governing permissions and limitations under the License.
 import { applyMiddleware, createStore } from 'redux';
 import createLogger from 'redux-logger';
 import rootReducer from '../reducers';
+import * as instrument from '../../shared/instrument';
+
+const instrumenter = store => next => action => {
```

Comment:
Well I warn you, if we do this more than once I'm going to wrap it in something easier on the eyes ;)